src/actions/coordinator.ts

```diff
@@ -2,29 +2,56 @@ import { Constructor, TinlakeParams } from '../Tinlake'
 
 export function CoordinatorActions<ActionsBase extends Constructor<TinlakeParams>>(Base: ActionsBase) {
   return class extends Base implements ICoordinatorActions {
-    solveEpoch = async () => {
```

Comment:
Is this functionality stable yet? If no, can we remove this file from the rewrite?

Reply:
The file is already in `master` as well: https://github.com/centrifuge/tinlake.js/blob/master/src/actions/coordinator.ts. It probably would have been semantically better to merge it later, but to speed up integration I immediately wanted to merge it into the rewrite. The changes to this file in this PR are only some comments.